docs/website/docs/command-reference/dev.md

```diff
@@ -244,6 +244,44 @@ The following command will override the `USER` Devfile variable with the `john`
 odo dev --var USER=john --var-file config.vars
 ```
 
+### Running on Podman
+
+Instead of deploying the container into a Kubernetes cluster, `odo dev` can leverage the podman installation on your system to deploy the container.
+
+You need to use the `--platform podman` flags to run the component using podman instead of a Kubernetes cluster.
+
+```console
+odo dev --platform podman
+```
+<details>
+<summary>Example</summary>
+
+```console
+$ odo init --name my-nodejs-app --devfile nodejs --starter nodejs-starter
```

Comment:
Maybe we can remove the call to `odo init` from this. We want to show an example output of `odo dev --platform podman`.

```suggestion
```